src/App.js

```diff
@@ -1,12 +1,235 @@
-import React from 'react';
+// PULL REQUEST AL AANGEMAAKT https://github.com/jelmer-agricola/frontend-homework-week-12-react-Fruitbasket-orderform/pull/1
+
+import React, {useState} from 'react';
 import './App.css';
 
+
+
 function App() {
-  return (
-    <>
-      <h1>Fruitmand bezorgservice</h1>
-    </>
-  );
+
+    const [bananas, setBananas] = useState(0);
+    const [strawberries, setStrawberries] = useState(0);
+    const [apples, setApples] = useState(0);
+    const [kiwis, setKiwis] = useState(0);
+
+
+    //    function reset fruit
+    function resetTotalFruit() {
+        setBananas(0);
+        setStrawberries(0);
+        setApples(0);
+        setKiwis(0);
+    }
+
+    // state niet in fruit counter maar in app
+
+    const [firstName, setFirstname] = useState('');
+    const [surName, setSurname] = useState('');
+    const [age, setAge] = useState(0);
+    const [zip, setZip] = useState('');
+    const [frequency, toggleFrequency] = useState('');
+    const [timeframe, toggleTimeFrame] = useState('');
+    const [comments, setComments] = useState('');
+    const [termsAndConditions, toggleTermsAndConditions] = useState(false);
+
+    function handleSubmit(e) {
+        e.preventDefault();
+        console.log(`
+            Voornaam: ${firstName}, 
+            Achternaam: ${surName}, 
+            Leeftijd: ${age}, 
+            Postcode: ${zip}, 
+            Bezorgfrequentie: ${frequency},
+            Opmerkingen: ${comments},
+            Algemene voorwaarden: ${termsAndConditions}
+    `);
+
+        console.log(`Uw bestelling:,
+            bananen: ${bananas},
+            appels: ${apples},
+            strawberries: ${strawberries},
+            kiwi's: ${kiwis},
+        `);
+        // console.log(bananas + " bananen", strawberries + " aardbeien", apples + " apples", kiwis + " kiwi's");
+
+        // console.log(firstName, surName, age, zip, frequency, timeframe, comments, termsAndConditions);
+    }
+
+    return (<>
+        <h1>Fruitmand bezorgservice</h1>
+
+
+        <article>
+            <h2> Bananen 🍌</h2>
+            {/* Button die zowel ervoor zorgt dat je niet boven 0 uitkomt en -1 kan doen. De disabled hoeft maar 1 keer op de button. Omdat je met de plusbutton alleen omhoog kan */}
+            <button type="button" disabled={bananas <= 0} onClick={() => setBananas(bananas - 1)}>
+                -
+            </button>
+            <p>{bananas}</p>
+            {/*button die ervoor zorgt dat bananen met 1 omhoog gaan*/}
+            <button type="button" onClick={() => setBananas(bananas + 1)}>
+                +
+            </button>
+
+        </article>
```

Comment:
Kijken naar alle componenten die in deze file gebruikt worden, dan zie ik veel overlap met dit stukje. Vaak is dat een indicatie om dat stukje los te trekken en een apart component van te maken. Dat stelt je in staat om te hergebruiken